lib/chef/knife/winrm_shared_options.rb

```diff
@@ -27,16 +27,16 @@ module WinrmSharedOptions
       # Shared command line options for knife winrm and knife wsman test
       def self.included(includer)
         includer.class_eval do
-          option :manual,
+          option :manual_list,
```

Comment:
needs to align with https://github.com/chef/chef/blob/master/lib/chef/knife/ssh.rb#L56